tests/functional/cylc-clean/00-basic.t

```diff
@@ -44,76 +40,60 @@ __FLOW__
 
 FUNCTIONAL_DIR="${TEST_SOURCE_DIR_BASE%/*}"
 # -----------------------------------------------------------------------------
 TEST_NAME="run-dir-readlink-pre-clean"
 readlink "$SUITE_RUN_DIR" > "${TEST_NAME}.stdout"
 
-cmp_ok "${TEST_NAME}.stdout" <<< "${TEST_DIR}/${SYM_NAME}-run/cylc-run/${SUITE_NAME}"
+cmp_ok "${TEST_NAME}.stdout" <<< "${TEST_DIR}/sym/run/cylc-run/${SUITE_NAME}"
 
 
-INSTALL_LOG_FILE=$(ls "${TEST_DIR}/${SYM_NAME}-log/cylc-run/${SUITE_NAME}/log/install")
+INSTALL_LOG_FILE=$(ls "${TEST_DIR}/sym/log/cylc-run/${SUITE_NAME}/log/install")
 TEST_NAME="test-dir-tree-pre-clean"
-tree --noreport --charset=ascii "${TEST_DIR}/${SYM_NAME}-"* > "${TEST_NAME}.stdout"
-
-# If rose-cylc plugin is installed add install files to tree.
-export ROSE_FILES=''
-python -c "import cylc.rose" > /dev/null 2>&1 &&
-    export ROSE_FILES="
-                    |-- opt
-                    |   \`-- rose-suite-cylc-install.conf
-                    |-- rose-suite.conf"
-
+run_ok "${TEST_NAME}" tree --noreport --charset=ascii "${TEST_DIR}/sym/"*"/cylc-run/${CYLC_TEST_REG_BASE}"
 # Note: backticks need to be escaped in the heredoc
 cmp_ok "${TEST_NAME}.stdout" << __TREE__
-${TEST_DIR}/${SYM_NAME}-cycle
-\`-- cylc-run
-    \`-- cylctb-${CYLC_TEST_TIME_INIT}
-        \`-- ${FUNCTIONAL_DIR}
-            \`-- cylc-clean
-                \`-- ${TEST_NAME_BASE}
-                    \`-- share
-                        \`-- cycle
-${TEST_DIR}/${SYM_NAME}-log
-\`-- cylc-run
-    \`-- cylctb-${CYLC_TEST_TIME_INIT}
-        |-- ${FUNCTIONAL_DIR}
-        |   \`-- cylc-clean
-        |       \`-- ${TEST_NAME_BASE}
-        |           \`-- log
-        |               \`-- install
-        |                   \`-- ${INSTALL_LOG_FILE}
-        \`-- leave-me-alone
-${TEST_DIR}/${SYM_NAME}-run
-\`-- cylc-run
-    \`-- cylctb-${CYLC_TEST_TIME_INIT}
-        \`-- ${FUNCTIONAL_DIR}
-            \`-- cylc-clean
-                \`-- ${TEST_NAME_BASE}
-                    |-- _cylc-install
-                    |   \`-- source -> ${TEST_DIR}/${SUITE_NAME}
-                    |-- flow.cylc
-                    |-- log -> ${TEST_DIR}/${SYM_NAME}-log/cylc-run/${SUITE_NAME}/log${ROSE_FILES}
-                    |-- share -> ${TEST_DIR}/${SYM_NAME}-share/cylc-run/${SUITE_NAME}/share
-                    \`-- work -> ${TEST_DIR}/${SYM_NAME}-work/cylc-run/${SUITE_NAME}/work
-${TEST_DIR}/${SYM_NAME}-share
-\`-- cylc-run
-    \`-- cylctb-${CYLC_TEST_TIME_INIT}
-        \`-- ${FUNCTIONAL_DIR}
-            \`-- cylc-clean
-                \`-- ${TEST_NAME_BASE}
-                    \`-- share
-                        \`-- cycle -> ${TEST_DIR}/${SYM_NAME}-cycle/cylc-run/${SUITE_NAME}/share/cycle
-${TEST_DIR}/${SYM_NAME}-work
-\`-- cylc-run
-    \`-- cylctb-${CYLC_TEST_TIME_INIT}
-        \`-- ${FUNCTIONAL_DIR}
-            \`-- cylc-clean
-                \`-- ${TEST_NAME_BASE}
-                    \`-- work
+${TEST_DIR}/sym/cycle/cylc-run/${CYLC_TEST_REG_BASE}
```

Comment:
Move the tree output down a couple of levels to avoid making the tests dependent on the registration base.